### PR TITLE
Implement i18n for English and French support

### LINKED ISSUE #11: Implement i18n for Multi-Language Support (French & English)
The project should support both English and French. Aspects of the project, including the frontend and backend, should be translated properly based on the user's language preference.

To implement `i18n` effectively, we need to take a few steps, such as:

- Updating the database to support multiple languages
- Creating language files and adding translations for each language
- Using the Laravel translation functions in the codebase to dynamically switch between the languages
- Updating the views with appropriate language tags, so that content is displayed in the correct language based on user preference

### PR DESCRIPTION
**Pull request by ExcelDurant**
This PR includes the addition of English and French translations for the initial messages in a Laravel project. The `lang` directory was created to house the translations for the project in two different languages: `en` and `fr`. 

The initial directories and translations were generated by running the `php artisan lang:publish` command. The French translations were then added to the appropriate files. 

closes #11

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -96,7 +96,7 @@
     |
     */
 
-    'fallback_locale' => 'en',
+    'fallback_locale' => 'fr',
```
Comment (Fenn-CS) [inline config/app.php:99]: Why?

**Review by Fenn-CS — CHANGES_REQUESTED** [review]
There should probably be a place in the header to make the language switch.

**Review by Fenn-CS — CHANGES_REQUESTED** [review]
Please let's use translation strings. Managing these keys usually grow out of hand.

See : https://laravel.com/docs/10.x/localization#using-translation-strings-as-keys

Also one should be able to use the button in the header to switch languages

File: lang/en/passwords.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    | Password Reset Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are the default lines which match reasons
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'reset' => 'Your password has been reset.',
```
Comment (Fenn-CS) [inline lang/en/passwords.php:16]:
```suggestion
    'Your password has been reset.' => 'Your password has been reset.',
```

I mentioned in a previous review that you should use the translation strings as the keys. Do this everywhere.


https://laravel.com/docs/10.x/localization#using-translation-strings-as-keys

**Review by Fenn-CS — CHANGES_REQUESTED** [review]
See inline

File: lang/en/passwords.php (hunk 2 above)
Comment (Fenn-CS) [reply lang/en/passwords.php]: Search if you can regenerate them in the particular way discussed or if you can copy them from somewhere. 

Finish this so translation work can proceed.

File: lang/en/home.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+    /*
+    |--------------------------------------------------------------------------
+    | Home Language Lines
+    |--------------------------------------------------------------------------
+    |
+    |
+    */
+
+    'our helping' => 'Our Helping',
```
Comment (Fenn-CS) [inline lang/en/home.php:13]:
```suggestion
    'Our helping' => 'Our Helping',
```


Or is it required to be in lowercase ?

**Review by Fenn-CS — COMMENTED** [review]
Have you tested this? Did the translation work with the default messages? for example the ones on auth forms?

**Comment by ExcelDurant** [comment]
> Have you tested this? Did the translation work with the default messages? for example the ones on auth forms?

The French validation translations seem not to work with translation strings. I am still looking for a solution to this issue

**Review by Fenn-CS — APPROVED** [review]

**Pull request merged**